specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -928,6 +1007,172 @@
+          "description": "Private endpoint state for site recovery.",
+          "enum": [
+            "None",
+            "Enabled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "VaultPrivateEndpointState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "IdentityData": {
+      "description": "Managed Identity Data, as received from ARM headers.",
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "description": "The tenant Id header.",
+          "type": "string"
+        },
+        "principalId": {
+          "description": "The principal Id header.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The Identity type, can be SystemAssigned/UserAssigned/None.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateEndpointConnectionVaultProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnection"
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "enum": [
+            "Succeeded",
+            "Deleting",
+            "Failed",
+            "Pending"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "privateEndpoint": {
+          "$ref": "#/definitions/PrivateEndpoint"
+        },
+        "privateLinkServiceConnectionState": {
+          "$ref": "#/definitions/PrivateLinkServiceConnectionState"
+        }
+      }
+    },
+    "PrivateEndpoint": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        }
+      }
+    },
+    "PrivateLinkServiceConnectionState": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "enum": [
+            "Pending",
+            "Approved",
+            "Rejected",
+            "Disconnected"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PrivateEndpointConnectionStatus",
+            "modelAsString": true
+          }
+        },
+        "description": {
+          "type": "string"
+        },
+        "actionsRequired": {
+          "type": "string"
+        }
+      }
+    },
+    "PrivateLinkResources": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next chunk of the response",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateLinkResource": {
```

Comment:
@deymadhumanti , please have description and readonly property for this as well?